waffle_hub/__init__.py

```diff
@@ -141,13 +141,17 @@ def __repr__(self):
 
 
 class DataType(BaseEnum):
+    # TODO: map to same value
+
     YOLO = enum.auto()
-    ULTRALYTICS = YOLO
+    ULTRALYTICS = enum.auto()
 
```

Comment:
Why do two type receive different value? 

Is it easier to check a value with a single value rather than checking multiple value?